test_driver/transitions_perf_test.dart

```diff
@@ -0,0 +1,231 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert' show json;
+import 'dart:io' show sleep;
+
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart' hide TypeMatcher, isInstanceOf;
+
+import 'isolates_workaround.dart';
+
+// Demos for which timeline data will be collected using
+// FlutterDriver.traceAction().
+//
+// Warning: The number of tests executed with timeline collection enabled
+// significantly impacts heap size of the running app. When run with
+// --trace-startup, as we do in this test, the VM stores trace events in an
+// endless buffer instead of a ring buffer.
+//
+// These names must match the output of GalleryDemo.describe in
+// lib/data/demos.dart.
+const List<String> _profiledDemos = <String>[
+  'Shrine@study',
+  'Rally@study',
+  'Crane@study',
+  'Fortnightly@study',
+  'Bottom navigation@material',
+  'Buttons@material',
+  'Cards@material',
```

Comment:
Should we also profile one of the cupertino demos?

Reply:
I agree, I'll add one